src/client/steam/interfaces/apps.cpp

```diff
@@ -33,14 +33,18 @@ namespace steam
 		return "english";
 	}
 
+	// This is used for checking DLC too.
 	bool apps::BIsSubscribedApp(unsigned int appID)
 	{
-		return true;
+		return appID == 366842 ? std::filesystem::exists("zone/zm_common.xpak")	//zombies
+			: appID == 366841 ? std::filesystem::exists("zone/mp_common.xpak")	//multiplayer
+			: appID == 366840 ? std::filesystem::exists("zone/cp_common.xpak")	//campaign
+			: true;
```

Comment:
two things here: i recommend checking file existance not relative to the working directory, but the host module and caching the result is a must. like so:

```suggestion
		static const auto has_zombies = std::filesystem::exists(::utils::nt::library{}.get_path() / "zone/zm_common.xpak");
		static const auto has_multiplayer = std::filesystem::exists(::utils::nt::library{}.get_path() / "zone/mp_common.xpak");
		static const auto has_campaign = std::filesystem::exists(::utils::nt::library{}.get_path() / "zone/cp_common.xpak");
		return appID == 366842 ? has_zombies
			: appID == 366841 ? has_multiplayer
			: appID == 366840 ? has_campaign
			: true;
```

that way you can even get rid of the comments, as the variable names speak for themselves. 
You will have to include the nt util tho:

`#include <utils/nt.hpp>`

Reply:
I changed it to use `::utils::nt::library{}.get_path()` like you said here and now it detects all of them as not installed. `get_folder()` seems to work though